src/index.html

```diff
@@ -17,11 +17,417 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="images\logo.png"
```

Comment:
The src attribute for the image uses a backslash (\) instead of a forward slash (/). This might cause issues in some environments. It's recommended to use forward slashes for file paths in HTML.